x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/60_behavioral_analytics_list.yml

```diff
@@ -1,7 +1,13 @@
 setup:
-  - skip:
-      version: "all"
-      reason: "@AwaitsFix https://github.com/elastic/elasticsearch/issues/95717"
+  # Ensure collections does not exists before running tests.
+  - do:
+      search_application.delete_behavioral_analytics:
+        name: my-test-analytics-collection
+        ignore: 404
+  - do:
+      search_application.delete_behavioral_analytics:
+        name: my-test-analytics-collection2
+        ignore: 404
 
```

Comment:
I don't understand why it's needed. The teardown section removes these collections so something is wrong here.